src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1495,8 +1494,8 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
         // With one of the precision mouses, one click is always a multiple of 120,
         // but the "smooth scrolling" mode results in non-int values
-
-        double newValue = (_rowsToScroll * rowDelta) + (currentOffset);
+        const auto rowsToScroll{ _rowsToScroll == WHEEL_PAGESCROLL ? GetViewHeight() : _rowsToScroll };

```

Comment:
please add a comment explaining what `WHEEL_PAGESCROLL` does, and/or where it comes from

Reply:
Im like 50/50 on this; it’s a quick Ctrl-click away, and it’s a documented Win32 API, but also it’s weird and probably worth commenting

Reply:
addressed.